FAQ/fr.md

```diff
@@ -0,0 +1,127 @@
+
+### Est-ce qu'il y a une estimation de la date de sortie publique de Quaver?
+
+Nous n'avons pas d'estimation pour la date de sortie publique de Quaver.
+
+### Comment utiliser ma clé une fois que je la reçois?
+
+Pour utiliser votre clé, suivez [ces étapes](https://support.steampowered.com/kb_article.php?ref=5414-TFBN-1352)
+
+## Gameplay
+
+### Quels modes de jeu sont actuelement supportés par Quaver?
+
+Quaver supporte actuelement le 4 touches (4k) et le 7 touches (7k).
+
+### Est-ce que je peux importer des maps provenant d'autres jeux?
+
+Oui, Quaver supporte en ce momment les fichiers .osz et .sm. Pour les importer, glissez les fichiers à l'intérieur de la fenêtre de jeu.
+
+### J'ai beaucoup de maps sur un autre jeu, puis-je les importer toutes d'un coup?
+
+Il est possible d'importer toutes vos maps d'un coup en liant le fichier du jeu “.db” à l'intérieur du fichier “quaver.cfg”, qui se trouve à l'intérieur du dossier de Quaver.
+
+### Est-ce que je peux importer mes textures (skin) d'un autre jeu?
+
+Le jeu en lui même ne peut faire de conversion de texture, en revanche vous pouvez télécharger un convertisseur de texture .osk [ici](https://rhythmgamers.net/QBC/) avec un [tutoriel](https://www.youtube.com/watch?v=pWeLbx48NVI).
+
+### J'ai trouvé un bug, Où est-ce que je peux le signaler?Tout les signalements de bugs doivent être postés sur notre [GitHub issues](https://github.com/Quaver/Quaver/issues). Faites-en sorte de poster tout ce que vous trouver, mais merci de vérifier si ce problème n'a pas déjà été signalé pour éviter les doublons.
+
+### J'ai une proposition de fonctionnalité, où puis-je la poster?
+
+Comme pour les signalements de bugs, les propositions de fonctionnalité doivent être envoyées sur [GitHub issues](https://github.com/Quaver/Quaver/issues). Choisissez simmplement “Feature Request” quand vous créez la publication!
+
+### Comment puis-je réduire latence audio des hitsounds sur Linux?
+
+Sur les options Audio vous pouvez trouver les paramètres  “Audio Device Period” et “Audio Device Buffer Length”. La rêgle de base est: tant que vous baissez ces valeurs la latence baisse, mais l'utilisation du CPU et la probabilité de problèmes de son augmentent. Essayez de paramètrer la "Period" par tranche de 2 ms et la "Buffer Length" à 8 ms et relancez le jeu. S'il y a des problèmes de son, essayez d'aumenter la "Period" ou la "Buffer Length".Pour réduire les problèmes de son nous recommendons d'ouvrir le fichier `/etc/security/limits.conf` et d'y ajouter une ligne comme ceci:
+```
+votre_nom_d'utilisateur      -   rtprio      99
+```
+Remplacez `votre_nom_d'utilisateur` par votre nom d'utilisateur Linux. Redemarrez votre système pour que ces changements prennent effet.Si vous avez de latence augmentant pendant que vous jouez, ouvrez `/etc/pulse/default.pa`, trouvez une ligne où il est écrit:
```

Comment:
Space between _effet._ and _Si_

Si vous avez de latence augmentant pendant que vous jouez
should become
Si la latence augmente pendant que vous jouez